Scala/Scala.sublime-syntax

```diff
@@ -138,7 +138,7 @@ contexts:
           pop: true
         - match: '([a-zA-Z$_][a-zA-Z0-9$_]*)\s*:\s*'
           captures:
-            1: entity.name.variable variable.parameter
+            1: entity.name.parameter variable.parameter
```

Comment:
`entity.name` should be used here at all. That is for the definition of things like classes, function, etc. `variable.parameter` is the correct scope to use. I just won't cherry-pick this commit, and instead remove the `entity.name.variable` part.